ui-components/src/page_components/sign-in/FormSignInErrorBox.tsx

```diff
@@ -24,7 +24,11 @@ const FormSignInErrorBox = ({ networkStatus, errors, errorMessageId }: FormSignI
       )}
 
       {networkStatus.content?.error && Object.entries(errors).length === 0 && (
-        <ErrorMessage id={`form-sign-in-${errorMessageId}-error`} error={!!networkStatus.content}>
+        <ErrorMessage
+          id={`form-sign-in-${errorMessageId}-error`}
+          error={!!networkStatus.content}
+          className="block mt-0 leading-normal"
```

Comment:
<img width="520" alt="Screen Shot 2022-07-29 at 7 35 32 PM" src="https://user-images.githubusercontent.com/65367387/181865286-0161ef58-2634-4f68-aa01-251c0691d265.png">

You'll also need `text-red-700` here

Reply:
good catch! Thanks